tutorial/source/intro_part_ii.ipynb

```diff
@@ -87,7 +87,7 @@
    "outputs": [],
    "source": [
     "def deferred_conditioned_scale(measurement, *args, **kwargs):\n",
-    "    return pyro.condition(scale, data={\"measurement\": measurement})(*args, **kwargs)"
+    "    return pyro.condition(scale, data={\"measurement\": measurement}, *args, **kwargs)"
```

Comment:
Could you remove `*args, **kwargs` in favor of `guess` instead, so that the intended meaning of `deferred_conditioned_scale` is preserved?
```python
def deferred_conditioned_scale(measurement, guess):
    return pyro.condition(scale, data={"measurement": measurement})(guess)
```